### PR TITLE
fix: bug docarray pull

### PR DESCRIPTION
**Pull request by azayz**
---

- [ ] This PR references an open issue
- [ ] Test added
- [ ] Documentation adjusted

### CODE REVIEW

File: now/utils.py [hunk 1]
```diff
@@ -136,6 +136,7 @@ def get_info_hubble(user_input):
             'Your hubble account is not verified. Please verify your account to deploy your flow as admin.'
         )
     user_input.jwt = {'token': client.token}
+    user_input.user_name = response['data']['name']
```
Comment (florian-hoenicke) [inline now/utils.py:139]: we could change it to admin_name

**Review by florian-hoenicke — CHANGES_REQUESTED** [review]

**Review by jupyterjazz — APPROVED** [review]
LGTM!

let's have 'team-now' as a constant since it's used in so many places

**Review by IyadhKhalfallah — APPROVED** [review]
LGTM

File: now/common/detect_schema.py [hunk 2]
```diff
@@ -65,7 +66,7 @@
~            filter_field_candidates_to_modalities[field_name] = field_value.__class__
~
~    if len(index_field_candidates_to_modalities.keys()) == 0:
~        raise ValueError(
```
Comment (joschkabraun) [inline now/common/detect_schema.py]: Is it possible to use the previously used environment variables?

File: now/now_dataclasses.py [hunk 3]
```diff
@@ -45,6 +45,7 @@ class UserInput(BaseModel):
     deployment_type: Optional[str] = None
     secured: Optional[StrictBool] = None
     jwt: Optional[Dict[str, str]] = None
+    admin_name: Optional[str] = None
```
Comment (joschkabraun) [inline now/now_dataclasses.py:48]: Let's remove this attribute as it can be fetched from hubble SDK.

**Review by joschkabraun — CHANGES_REQUESTED** [review]

File: now/data_loading/data_loading.py [hunk 4]
```diff
@@ -38,7 +38,20 @@ def load_data(user_input: UserInput, data_class=None) -> DocumentArray:
         da = _extract_es_data(user_input=user_input, data_class=data_class)
     elif user_input.dataset_type == DatasetTypes.DEMO:
         print('⬇  Download DocumentArray dataset')
-        da = DocumentArray.pull(name=user_input.dataset_name, show_progress=True)
+        if (
+            'LOCAL_TESTING' in os.environ
```
Comment (joschkabraun) [inline now/data_loading/data_loading.py:42]: this environment variable

File: now/now_dataclasses.py (hunk 3 above)
Comment (azayz) [reply now/now_dataclasses.py:48]: I think we should still keep this, because this is the same behaviour as the JWT token, both info come from Hubble SDK.

**Pull request merged**